service/conf/application.conf

```diff
@@ -59,8 +59,11 @@ play.http {
   # Default to Routes in the root package (aka "apps" folder) (and conf/routes)
   #router = my.application.Router
   #TO allow more data in request body
+  errorHandler = modules.ErrorHandler
+  actionCreator = modules.OnRequestHandler
```

Comment:
#TO allow more data in request body need to come below "actionCreator = modules.OnRequestHandler"

Reply:
Done